src/main/kotlin/io/menu/prompts/ListPrompt.kt

```diff
@@ -46,7 +46,7 @@ interface ListPrompt<T> : Prompt<List<T>> {
             validationRules.regex?.let { regex -> items.any { !it.matches(regex) } } == true -> {
                 Errors.invalidRegex(
                     regex = validationRules.regex,
-                    items = items.filterNot { it matches validationRules.regex }
+                    items = items.filterNot { it matches validationRules.regex!! }
```

Comment:
The Kotlin K2 compiler (enabled in Komac) is smarter and can smart cast this to a non-null Regex. The Kotlin team are still working on rewriting the IntelliJ IDEA plugin to support the new compiler version which is why you would have had a warning. It should compile fine without this non-null assertion

Reply:
Yes, it was suggested by IntelliJ IDEA. Will revert this change.